act-rules-format.bs

```diff
@@ -195,6 +165,28 @@ If either one of these passed, the rule has passed. Only if both fail, does that
 </div>
 
 
+Rule Grouping {#grouping}
+===================
+
+In accessibility testing, there are often multiple ways to solve the same problem. For instance, in HTML tables, header cells can be indicated through the `scope` attribute, by using the `headers` attributes, or by using ARIA labels. All of these separate techniques could be described in one big rule. But this creates a large, and often difficult to maintain rule. To ensure rules are kept small and atomic, they SHOULD be put into a rule-group instead.
+
+To meet the accessibility requirement, only one rule in a rule-group has to pass. This way, for our table example, one could write three separate rules, one for scope, one for headers+id and one ARIA labels). If a table meets any of these rules, it automatically passes the group, and the failed results of the other rules can be ignored.
```

Comment:
Need to be consistent with the use of "rule-group." Later we do not hyphenate it. I don't think it need hyphenation.